packages/next/client/future/image.tsx

```diff
@@ -270,6 +272,35 @@ function handleLoading(
       onLoadingCompleteRef.current({ naturalWidth, naturalHeight })
     }
     if (process.env.NODE_ENV !== 'production') {
+      if (img.getAttribute('data-nfill')) {
+        //Fill mode runtime warnings
+        if (
+          !img.getAttribute('sizes') ||
+          img.getAttribute('sizes') === '100vw'
+        ) {
+          let widthViewportRatio =
+            img.getBoundingClientRect().width / window.innerWidth
+          if (widthViewportRatio < 0.6) {
+            warnOnce(
+              `Image with src "${src}" has no sizes prop, but doesn't take up the full width of the screen. Use the sizes prop (https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes) to improve page performance.`
```

Comment:
Can we offer a suggested value of `sizes` here? How will the user know which value to use for `sizes`?

Reply:
I'd like to be able to, but it's a little tricky. If the calculated ratio is 40%, I could have it say `You could try 'sizes="40vw"'` or something, but that seems pretty fragile, since it's not clear if that image will always be 40% of the screen, or it'll change a lot at different screen sizes (in which case a more complex sizes prop is required)

Reply:
We shouldn’t link to MDN, we should link to our own docs. And our own does need a lot of improvements to explain how to use sizes.